src/mlpack/tests/lsh_test.cpp

```diff
@@ -539,31 +539,28 @@ BOOST_AUTO_TEST_CASE(MultiprobeTest)
       }
     }
   }
-
   BOOST_REQUIRE(foundIncrease);
 }
 
 /**
  * Test: This is a deterministic test that verifies multiprobe LSH works
- * correctly. To do this, we generate a query , q1, that will end up
- * outside of the bucket of its nearest neighbors in C1. The neighbors will be
- * found only by searching additional probing bins.
- *
- * We verify this actually is the case.
- * 
- * Note: This test doesn't work, because of the effect the random offsets of LSH
- * have on the 1st level codes. We need to think of something better.
+ * correctly. To do this, we generate two queries, q1 and q2. q1 is hashed 
+ * directly under cluster C2, q2 is hashed in C2's center. 
+ * We verify that:
+ * 1) q1 should have no neighbors without multiprobe.
+ * 2) q1 should have neighbors only from C2 with 1 additional probe.
+ * 3) q2 should have all neighbors found with 3 additional probes.
  */
-/*
 BOOST_AUTO_TEST_CASE(MultiprobeDeterministicTest)
 {
   math::RandomSeed(std::time(NULL));
+
   // Generate known deterministic clusters of points.
   const size_t N = 40;
   arma::mat rdata;
   GetPointset(N, rdata);
 
-  const int k = N/2;
+  const int k = N/4;
```

Comment:
A couple more style comments, this should be `N / 4` ("horizontal whitespace"). :)